 crazys-wm-mod/cBrothel.cpp

```diff
@@ -3576,41 +3576,41 @@ void cBrothelManager::do_food_and_digs(sBrothel *brothel, sGirl *girl)
 	{
 		g_Girls.RemoveTrait(girl, "Masochist", true);
 		// `J` zzzzzz - needs better text
-		ss << girl->m_Realname << " seems to be getting over her Masochistic tendencies.";
+		ss << girl->m_Realname << " seams to have become tired of hurting people to get off and lost the \"Masocist\" trait.;

```

Comment:
seams are for joining fabrics, seems like http://grammarist.com/homophones/seam-vs-seem/ can be of some help :)